src/Grids/spectralelement.jl

```diff
@@ -138,11 +139,12 @@ mutable struct SpectralElementGrid2D{
     internal_surface_geometry::IS
     boundary_surface_geometries::BS
     enable_bubble::Bool
+    element_mask::EM
 end
 
```

Comment:
```suggestion
function set_mask!(space::AbstractSpace, mask::Field)
```
with `Spaces.horizontal_space(axes(mask)) == Spaces.horizontal_space(axes(sapce))`